### ISSUE TITLE
BUG - EXT - Accessibility on Sharing feedback 

### ISSUE THREAD

**Issue by carwelsh** [SAP/guided-answers-extension#488]
### Related Feature
Feature request: _issue number_

### Description
When user has entered feedback the Send button should have focus before the cancel button when tabbing through using keyboard navigation.

### Steps to Reproduce
1. Open GA
2. Navigate through tree using keyboard
3. Open provide feedback using keyboard
4. Tab through feedback form ( enter feedback) 

### Expected results

Send button is given focus when text has been inputed 

### Actual results

Cancel button is given focus 
### Screenshots
If applicable, add screenshots to help explain the problem.

#### Version/Components/Environment	
Add any other context about the problem here
OS:                      
- [ ] Mac OS
- [ ] Windows
- [ ] Other

### Problem
{describe the problem} 

### Fix
{describe the fix} 

### Why was it missed
{Some explanation why this issue might have been missed during normal development/testing cycle}  

### How can we avoid this
{if we don’t want to see this type of issues anymore what we should do to prevent}

**Comment by oliversberzs** [comment]
@ritarora @carwelsh fixed & merged
